src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutHandler.java

```diff
@@ -252,6 +233,44 @@ public OperationalResponse checkout(OperationalRequest request) {
 		resp.setConfirm(true);
 		return resp;
 	}
+
+	public ResponseEntity<?> totalFavesAndCheckouts(int userId) {
+		OperationalResponse confirm = new OperationalResponse();
+		if(userRepository.getUserCount(userId) > 0) {
+			int totalCheckouts = userRepository.getCheckoutCount(userId);
+			List<User> userFavorites = userRepository.getUserFavorites(userId);
```

Comment:
You can just do something like,

```
OperationalResponse confirm = new OperationalResponse();
List<User> userFavorites = userRepository.getUserFavorites(userId);
if(userFavorites.isEmpty()) {
    ....
}

```

This will save a database call made by `userRepository.getUserCount(userId)`